src/coreclr/vm/stubmgr.cpp

```diff
@@ -1657,7 +1668,13 @@ BOOL RangeSectionStubManager::DoTraceStub(PCODE stubStartAddress, TraceDestinati
 #ifdef DACCESS_COMPILE
         DacNotImpl();
 #else
+    #if defined(TARGET_ARM64) && defined(__APPLE__)
+        // On ARM64 Mac, we cannot put a breakpoint inside of ExternalMethodFixupPatchLabel
+        LOG((LF_CORDB, LL_INFO10000, "RSM::DoTraceStub: Skipping on M1\n"));
```

Comment:
This isn't just for "M1". These comments should really be "macOS" or "arm64-macOS". The "arm64" isn't really needed because that is their only architecture now anyways.

Reply:
This is true throughout the PR.